.gitignore

```diff
@@ -5,6 +5,7 @@
 /vendor
 /build
 composer.lock
+.idea/
```

Comment:
Why is this one needed?

Reply:
As most people use phpstorm, this is just annoying to have not in this file.
Doesnt hurt anyone else so a recommended addition in general.

Reply:
Ok, fine with me :+1: